src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityDeleteController.php

```diff
@@ -55,20 +56,31 @@ class EntityDeleteController extends Controller
      */
     private $authorizationService;
 
+    /**
+     * @var ServiceService
+     */
+    private $serviceService;
+
     /**
      * @param CommandBus $commandBus
+     * @param EntityService $entityService
```

Comment:
The doc block does not add any context as all parameters are typehinted explicitly

Reply:
They were already there and were incomplete so I fixed them, I agree we could remove them completely.